packages/cubejs-athena-driver/driver/AthenaDriver.js

```diff
@@ -13,10 +13,11 @@ class AthenaDriver extends BaseDriver {
       credentials: {
         accessKeyId: config.accessKeyId || process.env.CUBEJS_AWS_KEY,
         secretAccessKey: config.secretAccessKey || process.env.CUBEJS_AWS_SECRET,
+        sessionToken: config.sessionToken || undefined
```

Comment:
What issue do you try to solve with || undefined? If `sessionToken` is not defined in `config` it will be undefined anyway.